Ckan_API/Ckan_API.py

```diff
@@ -0,0 +1,37 @@
+from urllib.parse import urljoin
+
+import requests
+
+
+class CkanAPICall:
```

Comment:
Why is this `CkanAPICall` and not `CkanAPIClient`? Typically, a class name tends to be a noun and not a verb (well, I guess "call" can be a verb too... but I think you know what I mean)

Reply:
yeah! I forget that the class name should be a noun.